stl/inc/yvals_core.h

```diff
@@ -413,23 +420,26 @@
 #include <vcruntime.h>
 #include <xkeycheck.h> // The _HAS_CXX tags must be defined before including this.
 
-#define _STL_PRAGMA(PRAGMA)          _Pragma(#PRAGMA)
+// Note that _STL_PRAGMA is load-bearing;
+// it still needs to exist even once CUDA and ICC support _Pragma.
+#if defined(__CUDACC__) || defined(__INTEL_COMPILER)
+#define _STL_PRAGMA(PRAGMA) __pragma(#PRAGMA)
+#else
+#define _STL_PRAGMA(PRAGMA) _Pragma(#PRAGMA)
+#endif
+
 #define _STL_PRAGMA_MESSAGE(MESSAGE) _STL_PRAGMA(message(MESSAGE))
 #define _EMIT_STL_MESSAGE(MESSAGE)   _STL_PRAGMA_MESSAGE(__FILE__ "(" _CRT_STRINGIZE(__LINE__) "): " MESSAGE)
 
 // clang-format off
 #define _EMIT_STL_WARNING(NUMBER, MESSAGE) \

```

Comment:
Pre-existing: Why `1` instead of `true`? This line should be consistent with 437.

Reply:
Nevermind: Garbage in / Garbage out.